example/main.tf

```diff
@@ -0,0 +1,29 @@
+module "opg_aws_config_accountwide" {
+  source = "./terraform-aws-config/account"
+
+  providers = {
+    aws = aws.eu_west_1
+  }
+}
+
+module "opg_aws_config_eu_west" {
```

Comment:
I think one of your examples should demonstrate all of the input variables

Reply:
Added some better examples with all the input vars on one them and also using the bespoke rules overwrite we talked about :)